PurchasesTests/SwiftSources/Purchasing/PurchaserInfoHelperTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  PurchaserInfoHelperTests.swift
+//  PurchasesTests
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Nimble
+import XCTest
+@testable import Purchases
+
+class PurchaserInfoHelperTests: XCTestCase {
+
+    let dateFormatter = DateFormatter()
+    let transactionsFactory = TransactionsFactory()
+
+    let dict = [
+        "100_coins": [
```

Comment:
`"100_coins"` would be consumable, right? but the endpoint doesn't return them? if that's the case, we should maybe come up with something else that wouldn't be consumable, like
`"sword_of_1000_truths"` and `"super_mega_shield_thingy"`

Reply:
The endpoint returns both non-consumables and consumables. I will add that to the tests